### PR TITLE
feat: add uptime-monitor for metabase

### PR DESCRIPTION
**Pull request by henry410213028**
* add a discord bot to monitor metabase service

### CODE REVIEW

**Comment by david30907d** [comment]
cool, looks good to me just 2 things:
1. what do you think if we put `uptime-monitor` into different folder. Because it seems like independent project to metabase
2. any backup plan when our `uptime-monitor` goes down? what's the pros and cons between on-prem uptime monitor and `cloud-hosted` uptime monitor?

**Comment by david30907d** [comment]
but in general it looks good to me 👍

**Comment by david30907d** [comment]
btw @henry410213028 remember to resolve conflict before merging~

**Comment by henry410213028** [comment]
> cool, looks good to me just 2 things:
> 
> 1. what do you think if we put `uptime-monitor` into different folder. Because it seems like independent project to metabase
> 2. any backup plan when our `uptime-monitor` goes down? what's the pros and cons between on-prem uptime monitor and `cloud-hosted` uptime monitor?

Move `uptime-monitor` outside from metabase_server folder is a good idea, this script can monitor other services not only metabase.

Currently, there is no others backup plan, if the monitor is down, this bot will disappear from discord members.

What is `cloud-hosted` uptime-monitor you mentioned? Is a GCP-related service or third-party service?

**Comment by david30907d** [comment]
> 

FYI: https://geekflare.com/monitor-website-uptime/

**Comment by david30907d** [comment]
> Move uptime-monitor outside from metabase_server folder is a good idea, this script can monitor other services not only metabase.

@henry410213028 if that makes sense to you, then request me to review again once you're ready, sounds good?

File: data_team/uptime-monitor/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Uptime Monitor
+
+This is a discord bot that check service is alive.
+
+## Usage
+
+1. (Run once) Create a `config.json` file and provide a vaild discord bot token and channel id.
+
+2. Launch the service with `docker-compose up --build -d`.
```
Comment (david30907d) [inline data_team/uptime-monitor/README.md:9]: thx for documenting this up! 🎉

**Review by david30907d — APPROVED** [review]
LGTM~ thx!

**Pull request merged**